data/queenstack.json)

```diff
@@ -0,0 +1,22 @@
+
+ { "name": "Agada Queen",
+  "type": "personal",
+  "bio": "Data scientist **Business Analyst**. Technical writer"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/maquinbeaury",
+       "icon": "FaTwitter"
+       }
+       {
+      "name": "Follow me on Linkedin",
+      "url": "https://twitter.com/maquinbeaury",
+      "icon": "FaLinkedin"
+      }
```

Comment:
Add a **'Comma'** after   **' } '**

Reply:
I think you have to use linkedin  in your url